tests/unit/connector/test_http_input.py

```diff
@@ -276,6 +311,37 @@ def test_get_next_with_hmac_of_raw_message(self):
         connector_next_msg, _ = connector.get_next(1)
         assert connector_next_msg == expected_event, "Output event with hmac is not as expected"
 
+    def test_endpoint_has_credentials(self, create_credentials):
+        credential_file_path = create_credentials
+        mock_env = {"LOGPREP_CREDENTIALS_FILE": str(credential_file_path)}
+        with mock.patch.dict("os.environ", mock_env):
+            new_connector = Factory.create({"test connector": self.CONFIG}, logger=self.logger)
+            new_connector.pipeline_index = 1
+            new_connector.setup()
+            endpoint_config = new_connector.http_server.endpoints_config.get("/auth-json-secret")
+            print(endpoint_config.credentials)
+            assert endpoint_config.credentials.username
+            assert endpoint_config.credentials.password
+
+    def test_endpoint_has_basic_auth(self, create_credentials):
+        credential_file_path = create_credentials
+        mock_env = {"LOGPREP_CREDENTIALS_FILE": str(credential_file_path)}
+        with mock.patch.dict("os.environ", mock_env):
```

Comment:
```suggestion
    @pytest.mark.parametrize("endpoint", ["/auth-json-secret", "/.*/[A-Z]{2}/json$"])
    def test_endpoint_has_credentials(self, endpoint, credentials_file_path):
        mock_env = {"LOGPREP_CREDENTIALS_FILE": credentials_file_path}
        with mock.patch.dict("os.environ", mock_env):
            new_connector = Factory.create({"test connector": self.CONFIG}, logger=self.logger)
            new_connector.pipeline_index = 1
            new_connector.setup()
            endpoint_config = new_connector.http_server.endpoints_config.get(endpoint)
            assert endpoint_config.credentials.username, endpoint
            assert endpoint_config.credentials.password, endpoint

    def test_endpoint_has_basic_auth(self, credentials_file_path):
        mock_env = {"LOGPREP_CREDENTIALS_FILE": credentials_file_path}
```

* make usage of `credentials_file_path` fixture to get rid of redefining symbol linter error
* parametrize the test to ensure regex endpoints can be used